### PR TITLE
MOSIP-34236: API development - getOriginalPartnerCertificate()

### PR DESCRIPTION
**Pull request by SwethaKrish4**

### CODE REVIEW

File: partner/partner-management-service/src/main/java/io/mosip/pms/partner/controller/PartnerServiceController.java [hunk 1]
```diff
@@ -7,6 +7,7 @@
 import javax.validation.constraints.NotNull;
 
 import io.mosip.pms.partner.dto.CertificateDto;
+import io.mosip.pms.partner.response.dto.*;
```
Comment (mayuradesh) [inline partner/partner-management-service/src/main/java/io/mosip/pms/partner/controller/PartnerServiceController.java:10]: remove this import, import each file separately

File: partner/partner-management-service/src/main/java/io/mosip/pms/partner/service/PartnerService.java [hunk 2]
```diff
@@ -27,15 +27,7 @@
...
-import io.mosip.pms.partner.response.dto.APIkeyRequests;
-import io.mosip.pms.partner.response.dto.CACertificateResponseDto;
-import io.mosip.pms.partner.response.dto.EmailVerificationResponseDto;
-import io.mosip.pms.partner.response.dto.PartnerCertDownloadResponeDto;
-import io.mosip.pms.partner.response.dto.PartnerCertificateResponseDto;
-import io.mosip.pms.partner.response.dto.PartnerCredentialTypePolicyDto;
-import io.mosip.pms.partner.response.dto.PartnerResponse;
-import io.mosip.pms.partner.response.dto.PartnerSearchResponseDto;
-import io.mosip.pms.partner.response.dto.RetrievePartnerDetailsResponse;
+import io.mosip.pms.partner.response.dto.*;
```
Comment (mayuradesh) [inline partner/partner-management-service/src/main/java/io/mosip/pms/partner/service/PartnerService.java:30]: remove this way of import; import separately

File: partner/partner-management-service/src/main/java/io/mosip/pms/partner/service/impl/PartnerServiceImpl.java [hunk 3]
```diff
@@ -25,6 +25,7 @@
 import io.mosip.kernel.core.authmanager.authadapter.model.AuthUserDetails;
 import io.mosip.pms.common.response.dto.ResponseWrapper;
 import io.mosip.pms.partner.dto.CertificateDto;
+import io.mosip.pms.partner.response.dto.*;
```
Comment (mayuradesh) [inline partner/partner-management-service/src/main/java/io/mosip/pms/partner/service/impl/PartnerServiceImpl.java:28]: same

File: partner/partner-management-service/src/main/resources/application-dev.properties [hunk 4]
```diff
@@ -61,6 +61,7 @@ pms.cert.service.token.request.clientId=mosip-partner-client
 pmp.ca.certificaticate.upload.rest.uri=https://dev.mosip.net/v1/keymanager/uploadCACertificate
 pmp.partner.certificaticate.upload.rest.uri=https://dev.mosip.net/v1/keymanager/uploadPartnerCertificate
 pmp.partner.certificaticate.get.rest.uri=https://dev.mosip.net/v1/keymanager/getPartnerCertificate/{partnerCertId}
+pmp.partner.original.certificate.get.rest.uri=${mosip.api.internal.url}/v1/keymanager/getPartnerSignedCertificate/{partnerCertId}
```
Comment (mayuradesh) [inline partner/partner-management-service/src/main/resources/application-dev.properties:64]: add this in Test folder's properties file also

**Review by mayuradesh — CHANGES_REQUESTED** [review]
comments given

File: partner/partner-management-service/src/main/java/io/mosip/pms/partner/controller/PartnerServiceController.java (hunk 1 above)
Comment (SwethaKrish4) [reply partner/partner-management-service/src/main/java/io/mosip/pms/partner/controller/PartnerServiceController.java]: imported each file in separate

File: partner/partner-management-service/src/main/resources/application-dev.properties (hunk 4 above)
Comment (SwethaKrish4) [reply partner/partner-management-service/src/main/resources/application-dev.properties:64]: added the properties in test folder

**Review by mayuradesh — APPROVED** [review]

**Pull request merged**